dataland-frontend/src/components/forms/CreateGdvDataset.vue

```diff
@@ -0,0 +1,327 @@
+<template>
+  <Card class="col-12 page-wrapper-card p-3">
+    <template #title>New Dataset - GDV</template>
+    <template #content>
+      <div v-if="waitingForData" class="d-center-div text-center px-7 py-4">
+        <p class="font-medium text-xl">Loading GDV data...</p>
+        <em class="pi pi-spinner pi-spin" aria-hidden="true" style="z-index: 20; color: #e67f3f" />
+      </div>
+      <div v-else class="grid uploadFormWrapper">
+        <div id="uploadForm" class="text-left uploadForm col-9">
+          <FormKit
+            v-model="companyAssociatedGdvData"
+            :actions="false"
```

Comment:
![image](https://github.com/d-fine/Dataland/assets/104447806/a241a3fd-610c-4ef1-86c0-81531315762a)
Is this normal that no activities can be choosen?

Reply:
No this is not normal. This will be fixed once @AdrianBeyer has finished his sub-task of making the `options` property of fields in the UploadConfig.ts customizable.
This dropdown needs custom dropdown options, which are at the moment not implementable.

Reply:
![image](https://github.com/d-fine/Dataland/assets/104447806/da854ebd-9c54-42a7-8c6b-8002370a8a51)
The document could not be downloaded, strangely for other documents it works

Reply:
![image](https://github.com/d-fine/Dataland/assets/104447806/fcb52b20-dfd7-491e-92b4-54f7ff035323)
Currently uploading of a document without a comment is possible, which in the case of such multiple documents should not be possible

Reply:
Changed to required

Reply:
My comments regarding the documents are resolved, activity still open

Reply:
Activity is working, resolving this here